### ISSUE TITLE
Add VDB Point Support to vdb_print

### ISSUE THREAD

**Issue by danrbailey** [AcademySoftwareFoundation/openvdb#617]
The vdb_print binary is missing support for VDB Point grids. This would be super useful. Some recommended functionality:

* total/active/inactive point count
* attribute name, type, uniform, index, flags, compression information
* group name and count information
* whether the descriptor is shared (option to print out all descriptors which differ?)

I suggest that this is only implemented for ABI=6 and above as there are some new virtual functions that will make this easier.

**Comment by kubaroth** [comment]
Just a heads-up, as Nick suggested. I'd like to give this one a try.

**Comment by RonMordo** [comment]
Hi, can i work on the issue?

**Comment by kmuseth** [comment]
Yes, please, go ahead :)Cheers,KenOn Feb 5, 2024, at 9:00 AM, Ron_Mordo ***@***.***> wrote:﻿
Hi, can i work on the issue?

**Comment by aveljkovic** [comment]
Hi, I am a first-time contributor and I would love to take this on!
